docker/Dockerfile.nvidia

```diff
@@ -0,0 +1,92 @@
+##############################################################
+# README with base system requirements
+# https://ngc.nvidia.com/catalog/containers/hpc:pgi-compilers
+#
+# BUILD: docker build --file docker/Dockerfile.pgi --tag devito:nvidia.pgi .
```

Comment:
I would add a READMe to the docker directory for that so that it is easily accessible to users.